test/integration/images/sti-fake-onbuild/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM sti_test/sti-fake
+
+ONBUILD RUN touch /sti-fake/src/onbuild
+
+RUN touch /sti-fake/src/index.html
```

Comment:
@ewolinetz your problem is first line of the [`assemble`](https://github.com/openshift/source-to-image/blob/master/test/integration/scripts/.sti/bin/assemble#L3) script, which is:
```
cp -Rf /tmp/src /sti-fake/
```
You don't need to satisfy that operation, so just change your RUN command into:
```
RUN touch /sti-fake/src/index.html && \
    mkdir -p /tmp/src
```